scala-package/core/src/main/scala/org/apache/mxnet/Shape.scala

```diff
@@ -34,6 +34,7 @@ class Shape(dims: Traversable[Int]) extends Serializable {
   def size: Int = shape.size
   def length: Int = shape.length
   def drop(dim: Int): Shape = new Shape(shape.drop(dim))
+  def dropBack(dim : Int) : Shape = new Shape(shape.dropRight(dim))
```

Comment:
I don't think you should introduce this public API, also the name is not intuitive.

Reply:
Is there any use case if the user would like to drop some dimension from back?

Reply:
suggest to let the user handle as required

Reply:
Ok, they can use `slice` as an alternative, will drop this